internal/corazawaf/casesensitive.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2024 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build coraza.rule.case_sensitive_args_keys
```

Comment:
I think it deserves a line in the README under the `Build tags` section

Reply:
Something like the following?
```
* `case_sensitive_args_keys` - enables case-sensitive matching for ARGS keys, aligning Coraza behavior with RFC 3986 specification. It will be enabled by default in the next major version.
```
if 👍 I would push the line and merge it